pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -285,8 +265,12 @@ func buildLogsQuery(start, end, step int64, mq *v3.BuilderQuery, fields map[stri
 		query := fmt.Sprintf(queryTmpl, step, op, filterSubQuery, groupBy, having, orderBy)
 		return query, nil
 	case v3.AggregateOperatorNoOp:
-		queryTmpl := constants.LogsSQLSelect + "from signoz_logs.distributed_logs where %s %s"
-		query := fmt.Sprintf(queryTmpl, timeFilter, filterSubQuery)
+		queryTmpl := constants.LogsSQLSelect + "from signoz_logs.distributed_logs where %s %sorder by %s"
+		if orderBy == "" {
+			orderBy = "timestamp"
```

Comment:
You wanted to use `id` here?

Reply:
I can see where you are coming from but default is fine and the rows will appear as they were ingested.